action.yml

```diff
@@ -37,7 +37,7 @@ runs:
   using: 'node20'
   main: 'dist/restore/index.js'
   post: 'dist/save/index.js'
-  post-if: "success() || github.event.inputs.save-always"
+  post-if: "success() || github.event.inputs.save-always == 'true' || github.event.inputs.save-always == true"
```

Comment:
This is not the reason `save-always` is broken.

`github.event.inputs.save-always` is not a valid context access.

See https://github.com/actions/cache/pull/1325#discussion_r1636968780 and https://github.com/actions/cache/issues/1315#issuecomment-2163822166 for more details.